config/routes.rb

```diff
@@ -25,6 +25,8 @@
   get 'dashboard/uploads', to: 'dashboard#uploads', as: :activity
   get 'charts/uploads', to: 'charts#uploads', as: :uploads_chart
   get 'charts/records', to: 'charts#records', as: :records_chart
+  
+  get '/data', to: 'data#index'
```

Comment:
@Hurlyburly2 this is more a question since I'm newer to rails - what is the reason not to do resourceful routing since index is a default/included route? something like `resources :data, only: [:index]`

Reply:
@marlo-longley I was unsure of the actual differences myself (my Rails is kind of rusty!) so I checked with a coworker who's much more expert. Both methods create the same route, but `resources` establishes some helper functions and corresponds to a resource. In this case, we're serving up a pretty static page, and there is no model for Data to which CRUD stuff is being done.